algorithms/CPlusPlus/Dynamic-Programming/edit-distance.cpp

```diff
@@ -0,0 +1,71 @@
+// Given two strings 's1' and 's2' and below operations that can performed on 's1'.
+// 1.Insert
+// 2.Remove
+// 3.Replace
+//Find minimum number of edits (operations) required to convert ‘s1’ into ‘s2’.  
+
+// NOTE: All above operations are of equal cost
+// Input:   s1 = "car", s2 = "cars"
+// Output:  1
+// We can convert s1 into s2 by inserting a 's'.
+
+//Program using Dynamic Programming
+//TIME COMPLEXITY: O(m*n) 
+
+#include <iostream>
+using namespace std;
+ 
+// Utility function to find the minimum of three numbers
+int min(int x, int y, int z) { return min(min(x, y), z); }
+ 
+int editDistDP(string s1, string s2, int m, int n)
+{
+    int dp[m + 1][n + 1];
```

Comment:
C++ lords would be unhappy when they see this.
If possible, kindly use dynamic array or an stl container for the same ( `std::vector` ) instead of VLA.

Reply:
Please have a look, I have made the specified changes.